Fix stall grapher x axis label.

wddgit opened the pull request:
#### PR description:

Since some recent changes, the stall grapher has been plotting
milliseconds not seconds (the x axis label says seconds). Make it
really plot seconds. Also add new command line arguments to
explicitly set the x axis limits in the stall grapher.

#### PR validation:

Nothing runs this Python script in the release or unit/RelVal tests. The
change has no effect other than on the plots in the PDF file produced
by the stall grapher.

cmsbuild commented:
The code-checks are being triggered in jenkins.

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-28206/12315

- This PR adds an extra 20KB to repository

cmsbuild commented:
A new Pull Request was created by  @wddgit (W. David Dagenhart) for master.

It involves the following packages:

FWCore/Concurrency


@cmsbuild, @smuzaffar, @Dr15Jones can you please review it and eventually sign? Thanks.
@makortel, @Martin-Grunewald this is something you requested to watch as well.
@davidlange6, @slava77, @fabiocos you are the release manager for this.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

wddgit commented:
please test

Nothing actually runs this code in any test...

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: 0d1eef83ac62a9e1cca21fddcd06e7ed3b917999
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-b39ea8/3016/summary.html

cmsbuild commented:
Comparison job queued.

Dr15Jones commented:
I’d rather have the values converted to seconds

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-b39ea8/3016/summary.html

Comparison Summary:
 - No significant changes to the logs found
 - Reco comparison results: 0 differences found in the comparisons
 - DQMHistoTests: Total files compared: 34
 - DQMHistoTests: Total histograms compared: 2961064
 - DQMHistoTests: Total failures: 1
 - DQMHistoTests: Total nulls: 0
 - DQMHistoTests: Total successes: 2960722
 - DQMHistoTests: Total skipped: 341
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: 0.0 KiB( 33 files compared)
 - Checked 147 log files, 16 edm output root files, 34 DQM output files

fabiocos commented:
@wddgit do you plan to further update the PR following the advice https://github.com/cms-sw/cmssw/pull/28206#issuecomment-543409687 by @Dr15Jones ?

wddgit commented:
Yes. I'll probably submit a new  update today. I keep getting side tracked on other things...

wddgit committed:
Fix bug in stall grapher utility

Fix bug in stall grapher utility. Was plotting microseconds,
but x axis labels said seconds. Make it plot seconds.
Also add new command arguments to set the x axis lower
and upper limits. This should not affect anything other
than the plots in the PDF file created by the stall grapher.

cmsbuild commented:
The code-checks are being triggered in jenkins.

cmsbuild commented:
+code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-28206/12568

- This PR adds an extra 20KB to repository

cmsbuild commented:
Pull request #28206 was updated. @cmsbuild, @smuzaffar, @Dr15Jones can you please check and sign again.

wddgit commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: acc0cb0cbf0cd47d9781dab69fdcfe5200f1dedf
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-b39ea8/3281/summary.html

cmsbuild commented:
Comparison job queued.

Dr15Jones commented:
+1

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next master IBs (tests are also fine). This pull request will now be reviewed by the release team before it's merged. @davidlange6, @slava77, @smuzaffar, @fabiocos (and backports should be raised in the release meeting by the corresponding L2)

fabiocos commented:
+1
